c2cgeoportal/lib/email_.py

```diff
@@ -34,13 +34,23 @@
 from email.Utils import formatdate
 
 
-def send_email(from_addr, to_addrs, body, subject, smtp_server):  # pragma: no cover
+def send_email(from_addr, to_addrs, body, subject, smtp):  # pragma: no cover
     msg = MIMEMultipart()
     msg["From"] = from_addr
     msg["To"] = ", ".join(to_addrs)
     msg["Date"] = formatdate(localtime=True)
     msg["Subject"] = subject
     msg.attach(MIMEText(body, "plain", "utf-8"))
-    smtp = smtplib.SMTP(smtp_server)
+
+    # Connect to server
+    if smtp.get('ssl', false):
```

Comment:
Use `False` instead of `false` (also for the lines above)

Reply:
And the value will be a string, so even if the value is `"False"` in the config file, the test will be true

Reply:
In the config file it should be a boolean!

Reply:
@sbrunner, @fredj  do you mean I have to change something in the vars_xx.yaml file?